landing-page/types/headnode/content/data/bookmarks.yaml

```diff
@@ -0,0 +1,5 @@
+# bookmarks:
+#   # Bookmarks/common directories to include in Flight File Manager
+#   - path: ""
+#     text: ""
+#     fa_icon: ""
```

Comment:
We'll need an `id` attribute for future functionality, we should include it here to ease any future upgrade path.